Makefile

```diff
@@ -0,0 +1,7 @@
+all: syntax
```

Comment:
Can we avoid having a Makefile? Could we put the Markdown linting as part of `npm run lint` (without using the ruby gem) and then just put `npm run lint` as the script of travis?